config/src/config/mempool_config.rs

```diff
@@ -120,7 +120,7 @@ impl Default for MempoolConfig {
             default_failovers: 1,
             enable_intelligent_peer_prioritization: true,
             shared_mempool_peer_update_interval_ms: 1_000,
-            shared_mempool_priority_update_interval_secs: 600, // 10 minutes (frequent reprioritization is expensive)
+            shared_mempool_priority_update_interval_secs: 100, // 10 minutes (frequent reprioritization is expensive)
```

Comment:
comment doesn't match the value

Reply:
These are just for forge tests. Will remove the config changes before landing.